edx_reverification_block/xblock/static/html/not_verified.html

```diff
@@ -0,0 +1,7 @@
+{% load i18n %}
+<div class="reverify-xblock">
+  <div class="reverify-not-verified">
+    <div class="msg-title">{% trans "Honor Track" %}</div>
+    <div class="msg-content">{% trans "You are taking this exam as an honor student." %}</div>
```

Comment:
Some learners who are working toward a verified certificate have to verify their identity before they can access this content. You do not have to verify your identity, and you can proceed to the next unit.

To learn more about verified certificates, see <a href="http://edx-guide-for-students.readthedocs.org/en/latest/SFD_certificates.html#verified-certificates”... Certificates</a> in the <i>EdX Guide for Students</i>.